coss/home/models.py

```diff
@@ -1,10 +1,34 @@
 from __future__ import absolute_import, unicode_literals
 
+from django.db import models
+
 from wagtail.wagtailcore.models import Page
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+from wagtail.wagtailcore.fields import RichTextField
+from wagtail.wagtailsnippets.models import register_snippet
 
 
 class GroupsGatheringsLandingPage(Page):
     pass
 
     class Meta:
         verbose_name = 'Groups and Gatherings Landing Page'
+
+
+@register_snippet
+class AboutSnippet(models.Model):
+    description = RichTextField(blank=True)
+    intro = RichTextField(blank=True)
+    why = RichTextField(blank=True)
```

Comment:
I would go for a more generic generic naming schema. These names don't make any sense for every project. Something like `header_text`, `body_text`, `image_text` etc make more sense. Feel free to choose something better from my suggestion. I just don't have a better idea atm.